grakn-engine/src/main/java/ai/grakn/engine/loader/MutatorTask.java

```diff
@@ -50,10 +55,10 @@
     private final QueryBuilder builder = Graql.withoutGraph().infer(false);
 
     @Override
-    public boolean start(Consumer<TaskCheckpoint> saveCheckpoint, TaskConfiguration configuration) {
+    public boolean start(Consumer<TaskCheckpoint> saveCheckpoint, TaskConfiguration configuration, BiConsumer<TaskState, TaskConfiguration> taskSubmitter) {
```

Comment:
@alexandraorth explained this to me and it looks pretty nasty! I would provide an actual interface here instead.